action/install.go

```diff
@@ -11,6 +11,10 @@ import (
 // Install describes an installation action
 type Install struct {
 	Driver driver.Driver // Needs to be more than a string
+
+	// OperationConfig is an optional handler that applies additional
+	// configuration to the operation before it is executed.
+	OperationConfig func(operation *driver.Operation)
```

Comment:
Did you consider abstracting out a single struct and then have specific action structs extend it? For example:
```
// ConfigurableAction is used to define actions which can configure operations before they are executed.
type ConfigurableAction struct {
    // OperationConfig is an optional handler that applies additional
    // configuration to an operation before it is executed.        
    OperationConfig OperationConfigFunc
}
```

Reply:
👍